src/utils/utils.cc

```diff
@@ -21,7 +21,79 @@
 #include "utils/utils.h"
 #include "utils/log.h"
 
+// Global string variables for each component
+const std::string INTEL_VID = "8087:";
```

Comment:
please fix the cpp lint issues below. use char instead of string when define global string constant.